### DISCUSSION TITLE
HP Smart

### DISCUSSION THREAD

**Discussion by giscus** [AwaitQuality/windowsonarm#69] [General]
# HP Smart

Check if HP Smart is ARM ready on Windows and can be used on Windows ARM devices such as the Surface with the Snapdragon X Elite.

https://windowsonarm.org/defba554-36bc-45af-8abb-7499864a3540

**Comment by Bo4ever90** [comment]
Does it allow remote printing without installing drivers for hp+ printers like the 9020 series?

**Comment by koderlink** [comment]
HP Smart v156.1.1125.0 still showing x64 architecture on Windows Task Manager

**Reply by OpenSource03** [reply]
Updated. Thanks.
